From f33a586497bcab8b7750994d840f67b72e04fd00 Mon Sep 17 00:00:00 2001
From: Rimon Xu <rimon.xu@rock-chips.com>
Date: Thu, 12 Apr 2018 08:53:58 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3328-evb-android: set cma memory to
 32M

When playing secure video, you need to interact with secure OS.
The interaction to secureOS must be a physical continuous memory,
so CMA needs to be used, and now CMA is not enough and 32M is
enough.

Change-Id: Ie665152808a048045e702b3e0e9d9137f30975b5
Signed-off-by: Rimon Xu <rimon.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3328-android.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328-android.dtsi b/arch/arm64/boot/dts/rockchip/rk3328-android.dtsi
index 3153314c8fd2..076200002d36 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3328-android.dtsi
@@ -62,6 +62,14 @@
 		ramoops_mem: ramoops@68000000 {
 			reg = <0x0 0x110000 0x0 0xf0000>;
 		};
+
+		/* global autoconfigured region for contiguous allocations */
+		linux,cma {
+			compatible = "shared-dma-pool";
+			reusable;
+			size = <0x0 0x2000000>;
+			linux,cma-default;
+		};
 	};
 
 	ramoops {
-- 
2.35.3

